lib/user.rb

```diff
@@ -0,0 +1,24 @@
+require 'pry'
+
+class User
+  attr_reader :name, :jokes
+  def initialize(name)
+    @name = name
+    @jokes = []
+  end
+
+  def learn(joke)
+    @jokes.push joke
+  end
+
+  def tell(user,joke)
+    user.learn joke
+  end
+
+  def joke_by_id(id)
```

Comment:
I would use `find` instead of `find_all` since we only need the *one* matching joke